e2e/addons/hello/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/julienschmidt/httprouter"
+)
+
+var (
+	sess = session.Must(session.NewSession())
+	svc  = dynamodb.New(sess)
+
+	tableName = os.Getenv("HELLO_TABLE_NAME")
+)
+
+// HealthCheck validates that the environment variables are available before we say the service is healthy.
+func HealthCheck(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	_, hasTableEnv := os.LookupEnv("HELLO_TABLE_NAME")
+	if !hasTableEnv {
+		log.Println("🚨HELLO_TABLE_NAME environment variable is missing!")
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	_, hasSecretEnv := os.LookupEnv("PASSWORD_SECRET")
+	if !hasSecretEnv {
+		log.Println("🚨PASSWORD_SECRET environment variable is missing!")
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	log.Println("🚑 healthcheck ok!")
+	w.WriteHeader(http.StatusOK)
+}
+
+func Hello(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	input := &dynamodb.PutItemInput{
+		Item: map[string]*dynamodb.AttributeValue{
+			"Name": {
+				S: aws.String(ps.ByName("name")),
+			},
+		},
+		TableName: aws.String(tableName),
+	}
+	if _, err := svc.PutItem(input); err != nil {
+		log.Printf("put item %v: %v\n", input, err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
```

Comment:
The tiniest nit in the entire world: `http.StatusCreated` :P

Reply:
lol fixed